test/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+add_subdirectory(util)
+
+file(GLOB NORMAL_TESTS "test_*.cpp")
+file(GLOB LARGE_TESTS "large_tests/*.cpp")
+file(GLOB FUZZY_TESTS "fuzz_*.cpp")
```

Comment:
Note that this change is a little dangerous -- a new test file will only be picked up after `cmake` has been run, not when `make check` is run in the build directory.